app/main.py

```diff
@@ -1,8 +1,63 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
```

Comment:
```suggestion
    def __init__(
            self,
            comfort_class: int,
            clean_mark: int,
            brand: str
        ) -> None:
```
it's better to structure it like this